R/load_fonts.R

```diff
@@ -8,24 +7,34 @@
 #' @export
 #'
 load_arcadia_fonts <- function(custom_font = "Suisse", fallback_font = "sans") {
-  # define font names to check for
-  font_names <- c("Suisse Int'l", "Suisse Int'l Semi Bold", "Suisse Int'l Medium", "Suisse Int'l Mono")
+  # check if custom font available
+  font_family <- custom_font
+
+  if (!("Suisse Int'l" %in% extrafont::fonts())) {
```

Comment:
is hardcoding here a problem? is there a way to use `custom_font` here instead? (since you removed hardcoding of `font_names`

Reply:
Font names are hardcoded, so I think it's okay to keep it this way for now... If font names change, we would have to update this script anyway.